docs/DeveloperGuide.md

```diff
@@ -112,6 +112,10 @@ Named Arguments:
 --ignore-standalone-config, -i
                      A flag to ignore the standalone config file in
                      the repo.
+--timezone UTC[±hh[mm]], -t UTC[±hh[mm]]
```

Comment:
`--timezone ZONE_ID[±hh[mm]], -t ZONE_ID[±hh[mm]]`, since it is no longer just `UTC` alone